### DISCUSSION TITLE
New

### DISCUSSION THREAD

**Discussion by AbdilleJr** [tailwindlabs/tailwindcss#14030] [Help]
Hello, am new tech and just discovered of tailwind would be great to get an pointers or advice as i start my journey with tailwind. Thanks

**Comment by wongjn** [comment]
Consider reading [the documentation website](https://tailwindcss.com/docs/utility-first). It contains all sorts of pointers and advice on using Tailwind CSS.

**Reply by AbdilleJr** [reply]
Thank you
